src/site/content/en/blog/codelab-stories-component/index.md

```diff
@@ -209,11 +312,32 @@ Last, we're using Javascript to toggle between our stories, why not give CSS a l
 }
 ```
 
-A keen eye would notice the `pointer-events: none` there and scratch their head. I'd say this is the only downside of the solution so far, is that a `.seen.story` is still on top and will receive taps, even though it's invisible. By setting the `pointer-events` to none, we turn the glass story into a window, and steal more user interactions. Not too bad of a trade off, not too hard to manage here in our CSS right now. We're not juggling `z-index`, I'm feeling good about this still.
+The `.seen` class will be added to a story that needs an exit.
+I got the custom easing function (`cubic-bezier(0.4, 0.0, 1,1)`)
+from Material Design's [Easing](https://material.io/design/motion/speed.html#easing)
+guide (see the *Accerlerated easing* section).
+
+{% Aside %}
+TODO(kayce): Explain the ampersand syntax
```

Comment:
we can push to the spec draft? https://drafts.csswg.org/css-nesting-1/#nest-selector

Reply:
Perfect